Source/Core/DolphinWX/ISOProperties/FilesystemPanel.cpp

```diff
@@ -23,6 +25,8 @@
 #include "Common/Logging/Log.h"
 #include "DiscIO/Enums.h"
 #include "DiscIO/Filesystem.h"
+// TODO: eww
```

Comment:
This (including the include right below) was supposed to be removed in one of the later commits. The removal must've gotten lost in the ISOProperties-to-FilesystemPanel rebase. I'll remove it later today.